src/Controller/Api/TagController.php

```diff
@@ -238,7 +238,9 @@ protected function handleWriteTag($request, $id = null)
             'csrf_protection' => false,
         ]);
 
-        FormHelper::removeFields($request->request->all(), $form);
+        if (class_exists(FormHelper::class)) {
```

Comment:
I think we need to find a solution for this.. @core23 you know what is the use for this remove?

Reply:
```php
    /**
     * This function remove fields available if there are not present in the $data array
     * The data array might come from $request->request->all().
     *
     * This can be usefull if you don't want to send all fields will building an api. As missing
     * fields will be threated like null values.
     */
    public static function removeFields(array $data, Form $form)
    {
        $diff = array_diff(array_keys($form->all()), array_keys($data));

        foreach ($diff as $key) {
            $form->remove($key);
        }

        foreach ($data as $name => $value) {
            if (!\is_array($value)) {
                continue;
            }

            self::removeFields($value, $form[$name]);
        }
    }
```

Reply:
Should I copy it to this controllers. WDYT?

Reply:
How many bundles use those methods?

Reply:
Only this.

Reply:
What about create the class (final) with only that method?